api/proto/info/info.proto

```diff
@@ -288,6 +309,9 @@ message ZInfoDevice {
   // eve
   uint32 rebootConfigCounter = 33;
   BootReason   last_boot_reason = 34;
+
+  // Cellular / LTE related information
+  ZCellularModuleInfo cellularInfo = 35;
```

Comment:
What do I send when there is no modem?
Multiuple modems and/or multiple SIMs for one modem?

"repeated" would be a first step to handle that. But multiple modems and/or SIMs might need a name to identify which modem/SIM the info is about.

Reply:
Yeah.. was thinking about multiple modems.. Was not sure if they exist in the edge servers or if the complexity is needed.

Hierarchy is:
   Modem --> Multiple Sims

We can make this:
   Multiple Modems -> Each modem with Multiple Sims.

Added a name each.

Reply:
No, no @kalyan-nidumolu -- as I pointed out -- SIMs have absolutely no relationship to the modems. The two are completely orthogonal in modern architectures -- this needs to be modeled.